python/examples/asset_agnostic_rules/requirements.txt

```diff
@@ -0,0 +1,2 @@
+python-dotenv
+/Users/benjaminnguyen/Code/sift/python
```

Comment:
This looks like it might be accidentally included?

Reply:
nice catch i was just about ot merge